operations/Dockerfile

```diff
@@ -1,59 +1,31 @@
 # syntax=docker/dockerfile:1.2
-FROM node:18-slim AS builder
+FROM node:20-slim AS base
 
 ENV PNPM_HOME="/pnpm"
 ENV PATH="$PNPM_HOME:$PATH"
+
 RUN corepack enable
 
+FROM base AS builder
```

Comment:
why the extra step?

there was FROM node:18-slim AS builder and now there is FROM base AS builder